prov/efa/src/efa_av.h

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright (c) 2013-2015 Intel Corporation, Inc.  All rights reserved.
```

Comment:
I'm guessing Git might be confused here due to the large change, but the diff is showing a removed Intel copyright - let's just make sure that's not actually the case

Reply:
Following up here: This particular line has existed in `efa_cm.c` since its [initial commit](https://github.com/ofiwg/libfabric/blob/0ecf26312ec07451d47d841553de7aba95b0040e/prov/efa/src/efa_cm.c) in 2019, yet FWICT, there's no code contributed by Intel in here. So this is probably safe to remove.